korge/src/jvmTest/kotlin/com/soywiz/korge/view/camera/CameraTest.kt

```diff
@@ -0,0 +1,87 @@
+package com.soywiz.korge.view.camera
+
+import com.soywiz.klock.*
+import com.soywiz.korge.*
+import com.soywiz.korge.input.*
+import com.soywiz.korge.time.*
+import com.soywiz.korge.tween.*
+import com.soywiz.korge.view.*
+import com.soywiz.korgw.*
+import com.soywiz.korim.color.*
+import com.soywiz.korma.geom.*
+import kotlinx.coroutines.*
+import kotlin.math.*
+
+fun main(): Unit = runBlocking { korge() }
+
+//TODO: Move completed sample to korge-samples
+suspend fun korge() = Korge(quality = GameWindow.Quality.PERFORMANCE, title = "Camera test", bgcolor = Colors.WHITE) {
+
+    onClick { speed = abs(speed - 1.0) }
+
+    fun elements() = Container().apply {
+        solidRect(400.0, 400.0, Colors.YELLOW)
+        solidRect(350.0, 350.0, Colors.YELLOWGREEN)
+        solidRect(300.0, 300.0, Colors.GREENYELLOW)
+        solidRect(250.0, 250.0, Colors.GREEN)
+        polygon(50.0, 7, Colors.DARKGREY).position(200.0, 100.0)
+    }
+
+    val cam = Camera(0.0, 0.0, 400.0, 400.0)
+    val cam2 = cam.copy()
+    val container = cameraContainer(400.0, 400.0, decoration = { solidRect(width, height, Colors.PINK) }) {
```

Comment:
CameraContainer on mac is only quarter the size it is supposed to have.
Probably related to #231 
<img width="1598" alt="Korge_und_korgeTemplate_–_main_kt__korgeTemplate_commonMain_" src="https://user-images.githubusercontent.com/5278352/86210905-b3c7b980-bb75-11ea-83d1-8cb3636ad638.png">